lib/src/bindings_player_ffi.dart

```diff
@@ -375,14 +375,18 @@ class FlutterSoLoudFfi {
   /// [volume] 1.0f full volume
   /// [pan] 0.0f centered
   /// [paused] 0 not pause
-  /// Return the handle of the sound, 0 if error
+  ///
+  /// Return the handle of the sound, [SoundHandle.error] if error
   SoundHandle play(
     SoundHash soundHash, {
     double volume = 1,
     double pan = 0,
     bool paused = false,
   }) {
     final handleId = _play(soundHash.hash, volume, pan, paused ? 1 : 0);
```

Comment:
It would be great if the error was propagated somehow from C++ land, because otherwise we have to return an unknown error, which is always frustrating to devs.

Reply:
There can only be 2 kinds of errors in the `play()` C function:
- the given `soundHash` is not found
- the `Soloud::play` cannot play an already loaded sound returning an `UNKNOWN_ERROR`
So, I can return a `PlayerError.soundHashNotFound` or `PlayerError.unknownError`. For the latter, I don't know what the cause could be.

Reply:
here the [Soloud.play()](https://solhsa.com/soloud/basics.html) doc (search `Soloud.play()` in the page)